src/builders.ts

```diff
@@ -1,8 +1,21 @@
 import { isAbsolute, relative } from 'path';
+import { createHash } from 'crypto';
 import slash = require('slash');
-import { DaysToDecay, LintMessage, LintResult, TodoConfig, TodoData } from './types';
+import {
+  DaysToDecay,
+  LintMessage,
+  LintResult,
+  Location,
+  Range,
+  TodoConfig,
+  TodoData,
+  TodoDataV1,
+  TodoDataV2,
+} from './types';
 import { getDatePart } from './date-utils';
 
+const LINES_PATTERN = /(.*?(?:\r\n?|\n|$))/gm;
```

Comment:
Is this just for eslint? We should make an issue (so we don't forget) to extract this and force it into a consumer problem

Reply:
Yep agreed. I think we ultimately want to have consumers pass us pre-normalized v2 format. I'll create an issue and follow that up in a separate PR.

Reply:
Logged #258 to capture this requirement.